### PR TITLE
update of math bot from: https://github.com/janagoetze/slurk-bots/tre…

### PR DESCRIPTION
**Pull request by sebag90**
this pull requests merges the changes dones on the [demo branch](https://github.com/janagoetze/slurk-bots/tree/demos/math)
Main changes:
* new task room layout with a background image, task description and some examples for the user
* user input is evaluated using the [ast module](https://docs.python.org/3/library/ast.html) instead of eval(). For questions, only ast.BinOps are accepted as good mathematical questions and answers are converted to floats. If the user input is malformed, the user will be notified before saving the question/answer.
* with the ```setup.py```script it is possible to quickly start slurk and the math bot for testing

This should probably be merged after the [wordle bot](https://github.com/clp-research/slurk-bots/pull/39)

### CODE REVIEW

**Comment by sebag90** [comment]
I'm putting this pull request on pause. I'll create a new one after merging some other updates for the math-bot from https://github.com/clp-research/slurk-bots/pull/33

**Comment by sebag90** [comment]
I'm reopening this after merging https://github.com/clp-research/slurk-bots/pull/40 into this branch for a more strict user input check by @wencke-lm

File: math/math_bot.py [hunk 1]
```diff
@@ -67,98 +74,151 @@ def new_task_room(data):
...
+        answer = re.sub(r"^answer\s*", "", cmd)
+        prop_solution = self._eval(answer, answer=True)
+
+        if room_id not in self.room_to_q:
             self.sio.emit(
                 "text",
-                {"message": f"The proposed answer is: {answer}",
-                 "room": room_id},
+                {
+                    "message": "Ups, no question found you could answer!",
```
[the remark this answers is not in the document]
Comment (sebag90) [reply math/math_bot.py]: Ups, I did it again.. fixed :+1:

**Review by janagoetze — APPROVED** [review]
@sebag90 please double-check that this is still up-to-date and then we can merge it

**Comment by sebag90** [comment]
looks good and is still working fine! I'm merging it

**Pull request merged**